packages/react-components/react-infobutton/etc/react-infobutton.api.md

```diff
@@ -36,15 +37,43 @@ export type InfoButtonSlots = {
 // @public
 export type InfoButtonState = ComponentState<InfoButtonSlots> & Required<Pick<InfoButtonProps, 'size'>>;
 
+// @public
+export const InfoLabel: ForwardRefComponent<InfoLabelProps>;
+
+// @public (undocumented)
+export const infoLabelClassNames: SlotClassNames<InfoLabelSlots>;
+
+// @public
+export type InfoLabelProps = ComponentProps<Partial<InfoLabelSlots>, 'label'> & Pick<InfoButtonProps, 'content'>;
```

Comment:
Open question: having a `content` prop on InfoLabel is potentially confusing (is it the label's content, or the infobutton's content?). 

Options are:
1. Leave it as-is:<br />`<InfoLabel content="info text">label text</InfoLabel>`
2. Rename it for the InfoLabel control only, to `info` or something (name suggestions?):<br />`<InfoLabel info="info text">label text</InfoLabel>`
    * Pro: Potentially less confusing name
    * Con: Name doesn't match InfoButton's name
3. Rename the prop for _both_ InfoButton and InfoLabel (name suggestions?)
    * Pro: Potentially less confusing name; and prop names match between InfoButton and InfoLabel
    * Con: Need to come up with a good name
4. Remove this prop entirely, and require use of the `infoButton` slot:<br />`<InfoLabel infoButton={{ content: "info text" }}>label text</InfoLabel>`
    * Pro: No redundant prop on InfoLabel
    * Con: Cumbersome API for the most important prop on InfoLabel

Reply:
To me option 3 makes more sense. I think the props should align since the InfoButton is still the main part of the component. As for the name, I think info is correct because that's what the "content" actually is. Maybe information to make it more verbose?

Reply:
After talking to @sopranopillow offline, another option came up, which might be ideal:

5. Change InfoButton so its children are the content of the popover. Currently, the `children` of InfoButton is the icon, which rarely gets overridden, and that could be moved to an `icon` slot.<br/>`<InfoButton>This is the info text</InfoButton>`<br/>
Then, InfoLabel could use the shorthand for its `infoButton` slot, and wouldn't require an extra prop. The slot could even be renamed to `info`.<br/>`<InfoLabel info="This is the info text">This is the label text</InfoLabel>`

Reply:
I think we'll go with the following: 
* Start with option 2 (rename the prop to `info` for InfoLabel only). 
* Then in a later PR, modify InfoButton itself with either option 3 (rename `content` to `info`), or option 5 (change `content` to be `children`).
* Subsequently, if we go with option 5, then we can remove the `info` prop on InfoLabel, and rename the `infoButton` slot to `info`. (Because of slot shorthands, that will make InfoLabel's API mostly unchanged for the common case: `info="This is the info text"` will continue to work as it did.)